### PR TITLE
[flutter_local_notifications] add ability to associate iOS user notifications with attachments

### PR DESCRIPTION
**Pull request by psipaylo**
This PR adds ability to pass attachments from Dart to the native iOS code and set them into UNNotificationContent of a new notification.

### CODE REVIEW

**Comment by psipaylo** [comment]
Hey @MaikuB , 

Sure thing, example app should be updated with demo of added functionality.

**Comment by MaikuB** [comment]
This looks good to me though I'm going to try make some tweaks to directly to this PR around comments and error handling prior to merging it in

File: flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m [hunk 1]
```diff
@@ -113,6 +116,16 @@ - (void)pendingNotificationRequests:(FlutterResult _Nonnull)result {
                 if (request.content.userInfo[PAYLOAD] != [NSNull null]) {
                     pendingNotificationRequest[PAYLOAD] = request.content.userInfo[PAYLOAD];
                 }
+                if (request.content.attachments != nil) {
```
Comment (MaikuB) [inline flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m]: Will be removing this as it's not actually being read from the Flutter/Dart side. It's also iOS specific

**Comment by psipaylo** [comment]
Thanks a lot for final adjustments and releasing it so fast!

**Comment by MaikuB** [comment]
No worries, I tend to release after it's merged for others to use ASAP :)

**Pull request merged**